koku/masu/api/db_performance/db_performance.py

```diff
@@ -140,21 +138,33 @@ def get_pg_settings(self, setting_names=None):
         cur = self._execute(sql, params or None)
         return cur.fetchall()
 
-    def get_pg_engine_version(self):
-        global SERVER_VERSION
-        if not SERVER_VERSION:
-            sql = """
--- PARSED PG ENGINE VERSION
-select (boot_val::int / 10000::int)::int as "release",
-       ((boot_val::int / 100)::int % 100::int)::int as "major",
-       (boot_val::int % 100::int)::int as "minor"
-  from pg_settings
- where name = 'server_version_num';
-"""
-            res = self._execute(sql, None).fetchone()
-            SERVER_VERSION.extend(res.values())
-
-        return SERVER_VERSION
+    def get_pg_engine_version(self) -> str:
+        sql = "SHOW server_version_num;"
+        res = self._execute(sql, None).fetchone()
+        server_version_num = res["server_version_num"]
```

Comment:
Instead of the `server_version_num`, we could just use the `server_version`. Then we don't need to do any parsing of the result at all.
```
postgres=# SHOW server_version;
         server_version         
--------------------------------
 14.9 (Debian 14.9-1.pgdg120+1)
```

Reply:
I looked at that but didn't know if we wanted the extra information at the end.

Reply:
I wasn't sure how consistent that output would be between versions. But we could go with a naive use of that. It would simplify things considerably.

Reply:
If you look here:
https://internal.console.redhat.com/api/cost-management/v1/db-performance/db-settings/
you can see the `server_version` which is just 14.9, so I doubt the extra stuff is even there.

Reply:
Interesting. I'm less sure about the output of `server_version` but I can give that a try.

Reply:
I switched to using `server_version`. I just made up some inputs for the test.